### ISSUE TITLE
"Passing more than one Security attribute is not supported" in AccessDecisionManager

### ISSUE THREAD

**Issue by gnat42** [scheb/2fa#34]
**Bundle version**: 5.1.3
**Symfony version**: 5.1.8

**Description**
I upgraded a project from SF4 to 5. I'm now getting this error when 2FA is enabled.

Passing more than one Security attribute to "Symfony\Component\Security\Core\Authorization\AccessDecisionManager::decide()" is not supported.

So far I've tracked it down to this. This seems to occur because in my security.yaml I have the following

```
        - { path: ^/practice/dashboard, roles: [ ROLE_UNVERIFIED, ROLE_USER ] }
```

If I remove one, it goes through fine. The error is coming from [this commit](https://github.com/symfony/symfony/commit/c64b0beffb858cdce2c4c5876af51c2a75bdfb1f)

There was a bug in SF with this reported [here](https://github.com/symfony/symfony/issues/36280)
with a fix [here](https://github.com/symfony/security-core/commit/79f27fbd467e48a324607d99747a86f1cfb3912e)

It seems like something needs to change in TwoFactorAccessDecider (around line 65). I haven't been able to figure out what other symfony components do in similar situations like this. If I disable 2FA my security.yml stuff works fine. So either this needs to use the symfony expression or multiple calls to decide as well. If I knew the correct solution I'd submit a PR (and still can if you guide me the way you'd want it solved).

**Comment by scheb** [comment]
Could you please post a stack trace how the call ends up in `Symfony\Component\Security\Core\Authorization\AccessDecisionManager::decide()`? Thanks!

**Comment by gnat42** [comment]
Of course

[scheb-2fa-trace.txt](https://github.com/scheb/2fa/files/5533181/scheb-2fa-trace.txt)

**Comment by scheb** [comment]
Confirmed. Seems to be a change in preparation for Symfony 6 when multiple attributes/roles are assigned to a route.

**Comment by scheb** [comment]
Bugfix released as v5.1.4.

**Comment by gnat42** [comment]
wow, thanks so much for looking and fixing this so quickly!
